CDP4IMEInstaller/CDP4LogInfoPlugin.wxs

```diff
@@ -14,11 +14,6 @@
         <File Id="fil_Manifest_CDP4LogInfo"  Vital="yes" Source="$(var.CDP4LogInfo.TargetDir)CDP4LogInfo.plugin.manifest" KeyPath="yes"></File>
       </Component>
 
-	    <!--<Component Id="comp_CDP4LogInfo_csvhelper" Guid="{873A7341-472F-4FA5-AC53-FA7EC8C2F102}" Location="local" Win64="$(var.Win64)">

```

Comment:
waarom is deze weg? volgens mij hebben we die voor de logger om naar csv te extorten